src/radical/pilot/staging_directives.py

```diff
@@ -141,17 +141,21 @@ def complete_url(path, context, log=None):
     certain locations, namely relative to
 
         * `client://`  : the client's working directory
-        * `resource://`: the RP    sandbox on the target resource
-        * `pilot://`   : the pilot sandbox on the target resource
-        * `task://`    : the task  sandbox on the target resource
+        * `endpoint://`: the root file system on the target resource
+        * `resource://`: the sandbox base dir on the target resource
```

Comment:
`file:///` and `pwd:///` are not described here.

Reply:
The semantics for those two schemas is clarified now.